setup_local.sh

```diff
@@ -34,13 +34,12 @@
 # ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-VARS_FILE=./.build_vars-`uname -s`.sh
+VARS_FILE=./.build_vars.sh
```

Comment:
Is this file any different from ./scons_local/utils/setup_local.sh ?

Reply:
Yes, unfortunately it is.  I could back out this part of the PR, but simply removing this file and using the alternative isn't that easy.

Reply:
ok, no problem.  Not worth it.